### ISSUE TITLE
More than on sequencer per mechanism 

### ISSUE THREAD

**Issue by Arutus** [tsunamayo/Starship-EVO#2762]
Hello I would like to suggest enabling the ability to attach more than one sequencer to a mechanism such as setting reset positions for turrets or folding/ transforming mechanisms.

**Comment by BrickMacklin** [comment]
Regarding multiple linked entities this is what I used it for. Half of my turrets were grandfathered with them before their removal while the over half do not. On the side that does after they are finished firing I can press a button on the bridge to restore them to their static position. Gif shows this below.
https://imgur.com/a/4X5O6QT

However for all the other turrets that now lack this ability they will forever be locked to their previous firing position. This looks silly for ships to fly around with their turrets all over the place. It would help us to be able to use this ability to set a static position on our entities.
![image](https://user-images.githubusercontent.com/37836252/89725721-575f8180-d9e0-11ea-958c-d3e8ad204669.png)

**Comment by Drillz007** [comment]
This is a good idea to kinda add on to this it might be good to have a operations protocol for AI to follow for its core or something as in if 
the AI is coming in for landing it must 
-reset turrets
-retract turrets
-deploy landing gear
-land
not saying every ship will need to reset its turrets but i can imagine some ships not being able to fit into some spaces to dock or something with the turret facing in the wrong direction

**Comment by tsunamayo** [comment]
yes I plan to do add an option for turret reset and also add a logic event system so you can trigger that folded position when the ship is landing for example. Thanks
